### PR TITLE
Quiz - Marshall Friskics-Warren

### PR DESCRIPTION
**Pull request by wmfwarren**
Added four elephant shaving products to my product page.

### CODE REVIEW

File: quiz.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+}
+
+/*Below are the sectionOne styles*/
+
+p.description {
+	min-height: 7em;
+}
+
+section.sectionOne p {
```
Comment (stevebrownlee) [inline quiz.css]: **Pro tip:** Don't name classes based on the sequence they have in the existing HTML.  What if, next month, your product owner or designer decides to change the order of the sections?  Make the class name reflective of the content that is in an element instead.

Change the name of `sectionOne`, `sectionTwo`, and `sectionThree`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+	</header>
+<div class="contentContainer">
+	<article>
+		<!-- Product 1 -->
+		<header >
+			<h1 class="prodTitle">Elephant Razor</h1> <!-- Product Name -->
+		</header>
+		<!-- Image section -->
+		<section class="sectionOne">
+			<img src="https://static.boker.de/us/images/zoom/140510.jpg" class="prodImg" alt="The Razor" width="80%" height="100px"> <!-- Product image -->
```
Comment (stevebrownlee) [inline index.html]: Use CSS to change the display properties of an `img` element instead of using the HTML attributes.

**Comment by stevebrownlee** [comment]
@wmfwarren Great job, Marshall.  I've added a couple of notes for improvement.  Make those changes and let me know when you push up again so I can review.

**Comment by wmfwarren** [comment]
I made changes to the section names to make them descriptive of the content. I also changed the img dimension parameters so their were in CSS and not in the  tag.

**Comment by stevebrownlee** [comment]
@wmfwarren 👍 

Merge it!

**Pull request merged**